### ISSUE TITLE
style.scss build failure when jekyll-now is in user submodule

### ISSUE THREAD

**Issue by anne-decusatis** [barryclark/jekyll-now#263]
I'm having what looks like the same issue as https://github.com/barryclark/jekyll-now/issues/69. It looks like that one was resolved by... both the people affected finding solutions other than using jekyll-now? 

To reproduce:
1. Fork jekyll-now, rename to what you want the subdirectory name to be, make config changes
2. in your username.github.io repo: git submodule add {https clone link for your forked repo}
3. commit and push your .io repo, get build failure email
optional:
4. copy _config.yml into root of username.github.io repo, it doesn't help
5. remove --- from beginning of style.scss, now build succeeds but there's no styles

let me know if you need any more info!

**Comment by anne-decusatis** [comment]
This failure also occurs in plain Jekyll. Might be a problem with Github or my setup.

**Comment by barryclark** [comment]
Did you get this working ann?

I'm confused as to why you're using `git submodule`?

**Comment by anne-decusatis** [comment]
No, I haven't found a solution :(

I have other things hosted on my github.io domain; ideally I'd like a Jekyll blog to be in one folder, but I still need to host other things in other folders. I thought that "git submodule" would be a good way to solve this problem but I'm open to other suggestions.

**Comment by cjerrington** [comment]
Would creating another Repo on your account and make the default "gh-pages" work the same? That way username.github.io would have its root for the website and "sub-folder" as a gh-pages branch would be username.github.io/sub-folder. I think this is what you are asking for as well...

**Comment by anne-decusatis** [comment]
thanks @cjerrington! I did figure out how to get the behavior I wanted eventually by doing that but I must have forgotten to update this issue.

To fix, for anyone else reading along:
Don't use a git submodule
Create a new repository on github.com, separate from your github.io repo
Your new repository should be your Jekyll blog - can be with Jekyll Now, though by the point I did it I was using plain Jekyll. The name of this repository will be the folder/subdirectory on your github.io for your Jekyll blog. 
Write new blog posts with commits on a branch of your new repository titled `gh-pages`. 
Github shows gh-pages branches as subdirectories of your github.io without any work on your end. 
your Jekyll `_config.yml` should have the `baseurl` as the subdirectory your blog is in == the name of your blog repository.
